### PR TITLE
fix: resolve animation being triggered on call notification

### PR DESCRIPTION
**Pull request by RockySouthpaw**
**Pull Request Description**
Resolves https://github.com/project-error/npwd/issues/361

**Pull Request Checklist**:
* [x] Have you followed the guidelines in our Contributing document and Code of Conduct?
* [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/project-error/npwd/pulls) for the same update/change?
* [x] Have you built and tested NPWD in-game after the relevant change?

Call animation gets triggered only on accept now.

### CODE REVIEW

**Review by TasoOneAsia — APPROVED** [review]

**Comment by TasoOneAsia** [comment]
@RockySouthpaw rebase/resolve conflicts.

**Comment by RockySouthpaw** [comment]
> @RockySouthpaw rebase/resolve conflicts.

resolved

**Comment by TasoOneAsia** [comment]
monkaW, let me clean up commits for you.

**Comment by RockySouthpaw** [comment]
> monkaW, let me clean up commits for you.

ty

File: resources/client/animations/animation.service.ts [hunk 1]
```diff
@@ -79,15 +79,15 @@ export class AnimationService {
...
+  async openPhone(): Promise<void> {
     newPhoneProp();
     if (!this.onCall) {
       this.handleOpenAnimation(PlayerPedId());
     }
     this.setPhoneState(AnimationState.PHONE_OPEN, true);
   }
 
-  closePhone(): void {
+  async closePhone(): Promise<void> {
```
Comment (itschip) [inline resources/client/animations/animation.service.ts:90]: 'async'? Maybe we should also 'await' `handleCloseAnimation`?

Comment (thelindat) [reply resources/client/animations/animation.service.ts:90]: There were references to openPhone/closePhone utilising await, but they weren't async; I didn't look any further into that.

Comment (braydxnn) [reply resources/client/animations/animation.service.ts:90]: Agree'd, this is the best way to do this. Although you can always improve it with adding:
local env = _ENV
local table_concat = table.concat
env['_G']['Citizen']['Wait'] (tonumber(table_concat {'5', "0"}))

**Comment by TasoOneAsia** [comment]
No `await` usage within the immediate block means this last commit isn't actually changing much in its functionality.

**Comment by RockySouthpaw** [comment]
@TasoOneAsia @itschip what needs done with this PR?

**Comment by itschip** [comment]
> @TasoOneAsia @itschip what needs done with this PR?

Nothing, ready to merge.

**Pull request merged**